### PR TITLE
Added User-Agent

### PR DESCRIPTION
**Pull request by Seggan**
This fixes a bug where `¨U` gets a 403 because of the wrong user agent.

### CODE REVIEW

**Review by Lyxal — APPROVED** [review]
Lgtm

File: vyxal/elements.py [hunk 1]
```diff
@@ -3178,7 +3178,11 @@ def replace_until_no_change(lhs, rhs, other, ctx):
 def request(lhs, ctx):
     """Element ¨U
     (str) -> Send a GET request to a URL if online"""
-    x = urllib.request.urlopen(urlify(lhs)).read()
+
+    req = urllib.request.Request(
+        urlify(lhs), headers={"User-Agent": "Mozilla/5.0 Vyxal"}
```
Comment (cgccuser) [inline vyxal/elements.py:3183]: Might want to use the same user agent as Chrome

**Review by cgccuser — CHANGES_REQUESTED** [review]

File: vyxal/elements.py (hunk 1 above)
Comment (Seggan) [reply vyxal/elements.py:3183]: All user agents [start with Mozilla](https://stackoverflow.com/questions/1114254/why-do-all-browsers-user-agents-start-with-mozilla)

**Review by chunkybanana — APPROVED** [review]

**Comment by Seggan** [comment]
@cgccuser u making Jyxal PRs but not even reviewing this one 😆

**Review by cgccuser — APPROVED** [review]
I’d prefer a bigger user agent including Chrome and Safari but whatever. This is overdue already

**Comment by cgccuser** [comment]
@Seggan Hey, I didn’t see it, and besides, I’ve been too busy with those Jyxal PRs :p

**Pull request merged**